backends/arm/arm_backend.py

```diff
@@ -63,14 +66,17 @@ def generate_ethosu_compile_spec(
 
 
 def generate_tosa_compile_spec(
+    permute_memory_to_nhwc: Optional[bool] = None,
```

Comment:
why is this a choice? i.e. don't we always want this to be true?

Reply:
i agree. my original thought was we could also permute to other memory_formats but this seems less useful.